libraries/collectd_service.rb

```diff
@@ -104,6 +104,13 @@ def action_enable
             notifies :restart, new_resource, :delayed
           end
 
+          # Installing package starts collectd service automatically
+          # Disable this so that collectd can be managed through poise-service
+          service new_resource.package_name do
+            action [:disable, :stop]
```

Comment:
[dpkg_autostart cookbook] (https://github.com/hw-cookbooks/dpkg_autostart) does not seem to work for collectd. I am not sure why. This logic will work, because service instance managed by poise is only started in the superclass and we call `super` at the very end of `action_enable` method in https://github.com/bloomberg/collectd-cookbook/blob/master/libraries/collectd_service.rb#L131. I tested this on ubuntu 14/12/10 and it works well. As far as handling it at poise level, this was already raised as https://github.com/poise/poise-service/issues/20 and considered as something that is not in scope of poise.